src/css/styles.scss

```diff
@@ -44,3 +40,490 @@ $blue: #4261ab;
 		color: $blue;
 	}
 }
+
+body {
+	background-image: url("../images/sunset.png");
+	background-repeat: no-repeat;
+	background-attachment: fixed;
+	background-size: cover;
+	background-position: center;
+	height: 100vh;
+}
+
+nav {
+	width: 440px;
+	height: 50px;
+	align-items: center;
+	margin-left: 30px;
+	margin-top: 15px;
+	background: rgba(153, 58, 157, 0.9);
+	border: 1px solid #000;
+	box-sizing: border-box;
+	border-radius: 40px;
+
+	@media screen and (min-width: $tablet-vertical) {
+		width: 650px;
+		height: 67px;
+		margin-top: 20px;
+		margin-left: 60px;
+	}
+
+	@media screen and (min-width: $desktop) {
+		width: 1030px;
+		height: 72px;
+		margin-top: 20px;
+		margin-left: 85px;
+	}
+}
+
+.reftoindex {
+	position: relative;
+	width: 241px;
+	height: 71px;
+	font-family: Jura, sans-serif;
```

Comment:
It looks like you're using this typeface a lot. Should it be on the body element so that it's being used as your default typeface for the whole site?